### ISSUE TITLE
Mining Ravencoin. Thanks for the answer.

### ISSUE THREAD

**Issue by TheIlyaSteep** [UnamSanctam/SilentCryptoMiner#2201]
https://github.com/UnamSanctam/SilentCryptoMiner/issues/2176
I took your comments into account and presented the settings below, will the miner be less noticeable?

Question, if I glue the miner and any other .exe file using yours - "UnamBinder 1.3.0". Will it encrypt the miner?
![1](https://github.com/UnamSanctam/SilentCryptoMiner/assets/42769532/b2721140-410f-4e62-8c42-78daf2bfa0c7)
![2](https://github.com/UnamSanctam/SilentCryptoMiner/assets/42769532/28519c0c-e65c-4af5-8e1b-98584245cc7d)
![3](https://github.com/UnamSanctam/SilentCryptoMiner/assets/42769532/9cbbac16-94ab-44ce-9dc5-3705b9c1d976)
![4](https://github.com/UnamSanctam/SilentCryptoMiner/assets/42769532/c1f5f09d-6fa8-4057-848d-b06bc4926429)

**Comment by UnamSanctam** [comment]
Yes it might be less noticeable, and using the UnamBinder will encrypt the files (miner and whatever else you bind) though the bound file will likely be detected.

**Comment by TheIlyaSteep** [comment]
How can a victim remove a miner?

**Comment by TheIlyaSteep** [comment]
What exactly does a watchdog timer do?

**Comment by UnamSanctam** [comment]
The watchdog constantly checks if the miner is correctly running and correctly installed, and if not then it fixes it https://github.com/UnamSanctam/SilentCryptoMiner/wiki#watchdog.
